[2025-05-22T14:28:37.185+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Datamart_Traffict_Accident.Load_Data scheduled__2025-05-21T15:45:00+00:00 [queued]>
[2025-05-22T14:28:37.203+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Datamart_Traffict_Accident.Load_Data scheduled__2025-05-21T15:45:00+00:00 [queued]>
[2025-05-22T14:28:37.204+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-22T14:28:37.206+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-22T14:28:37.207+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-22T14:28:37.228+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Load_Data> on 2025-05-21 15:45:00+00:00
[2025-05-22T14:28:37.235+0000] {standard_task_runner.py:52} INFO - Started process 1494 to run task
[2025-05-22T14:28:37.240+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Datamart_Traffict_Accident', 'Load_Data', 'scheduled__2025-05-21T15:45:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/Datamart_Traffict_Accident.py', '--cfg-path', '/tmp/tmpsgixvzlt', '--error-file', '/tmp/tmpto0sk5y1']
[2025-05-22T14:28:37.243+0000] {standard_task_runner.py:80} INFO - Job 11: Subtask Load_Data
[2025-05-22T14:28:37.364+0000] {task_command.py:371} INFO - Running <TaskInstance: Datamart_Traffict_Accident.Load_Data scheduled__2025-05-21T15:45:00+00:00 [running]> on host 2262c451ca30
[2025-05-22T14:28:37.454+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Datamart_Traffict_Accident
AIRFLOW_CTX_TASK_ID=Load_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-21T15:45:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-21T15:45:00+00:00
[2025-05-22T14:28:37.456+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-22T14:28:37.459+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Load_Datamart_Data.py']
[2025-05-22T14:28:37.471+0000] {subprocess.py:85} INFO - Output:
[2025-05-22T14:28:39.364+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-05-22T14:28:39.365+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-05-22T14:28:39.365+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-05-22T14:28:39.366+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-05-22T14:28:39.367+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-05-22T14:28:39.538+0000] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-22T14:28:39.630+0000] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-05-22T14:28:39.631+0000] {subprocess.py:92} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-05-22T14:28:39.634+0000] {subprocess.py:92} INFO - org.postgresql#postgresql added as a dependency
[2025-05-22T14:28:39.636+0000] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-f1c81aed-4be8-429a-831d-6717e5d65791;1.0
[2025-05-22T14:28:39.637+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-22T14:28:39.750+0000] {subprocess.py:92} INFO - 	found org.postgresql#postgresql;42.6.0 in central
[2025-05-22T14:28:39.775+0000] {subprocess.py:92} INFO - 	found org.checkerframework#checker-qual;3.31.0 in spark-list
[2025-05-22T14:28:39.789+0000] {subprocess.py:92} INFO - :: resolution report :: resolve 149ms :: artifacts dl 4ms
[2025-05-22T14:28:39.790+0000] {subprocess.py:92} INFO - 	:: modules in use:
[2025-05-22T14:28:39.790+0000] {subprocess.py:92} INFO - 	org.checkerframework#checker-qual;3.31.0 from spark-list in [default]
[2025-05-22T14:28:39.791+0000] {subprocess.py:92} INFO - 	org.postgresql#postgresql;42.6.0 from central in [default]
[2025-05-22T14:28:39.791+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-22T14:28:39.792+0000] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-22T14:28:39.793+0000] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-22T14:28:39.793+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-22T14:28:39.793+0000] {subprocess.py:92} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-05-22T14:28:39.794+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-22T14:28:39.795+0000] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-f1c81aed-4be8-429a-831d-6717e5d65791
[2025-05-22T14:28:39.795+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-22T14:28:39.798+0000] {subprocess.py:92} INFO - 	0 artifacts copied, 2 already retrieved (0kB/4ms)
[2025-05-22T14:28:39.995+0000] {subprocess.py:92} INFO - 25/05/22 14:28:39 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-22T14:28:40.258+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-05-22T14:28:40.259+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-05-22T14:28:40.260+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-05-22T14:28:49.966+0000] {subprocess.py:92} INFO - [Stage 4:>                                                          (0 + 1) / 1]                                                                                Saved: /opt/***/data/crash_time.csv
[2025-05-22T14:28:49.967+0000] {subprocess.py:92} INFO - Saved: /opt/***/data/crash_factors.csv
[2025-05-22T14:28:50.441+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-22T14:28:50.486+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Datamart_Traffict_Accident, task_id=Load_Data, execution_date=20250521T154500, start_date=20250522T142837, end_date=20250522T142850
[2025-05-22T14:28:50.532+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-22T14:28:50.585+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-22T14:28:50.625+0000] {dagrun.py:567} INFO - Marking run <DagRun Datamart_Traffict_Accident @ 2025-05-21 15:45:00+00:00: scheduled__2025-05-21T15:45:00+00:00, state:running, queued_at: 2025-05-22 14:27:46.269878+00:00. externally triggered: False> successful
[2025-05-22T14:28:50.628+0000] {dagrun.py:627} INFO - DagRun Finished: dag_id=Datamart_Traffict_Accident, execution_date=2025-05-21 15:45:00+00:00, run_id=scheduled__2025-05-21T15:45:00+00:00, run_start_date=2025-05-22 14:27:46.315915+00:00, run_end_date=2025-05-22 14:28:50.628339+00:00, run_duration=64.312424, state=success, external_trigger=False, run_type=scheduled, data_interval_start=2025-05-21 15:45:00+00:00, data_interval_end=2025-05-21 16:00:00+00:00, dag_hash=1d11901268865888d11875b6ae542d43
